src/api/table.ts

```diff
@@ -18,11 +18,14 @@ export const updateVenueTable = async ({
   subtitle,
   capacity,
 }: UpdateVenueTableProps) => {
+  const updatedCapacity = parseInt(capacity.toString(), 10);
+  const tableColumns = tableOfUser?.columns ?? 1;
   const updatedTable = {
     ...tableOfUser,
     title,
     subtitle,
-    capacity,
+    capacity: updatedCapacity,
+    rows: Math.ceil(updatedCapacity / tableColumns),
```

Comment:
Why `.ceil` and not `.round`? If `updatedCapacity / tableColumns` is `99.01` it will be rounded to `100` instead of `99` where `.round` will use the closer value.

Reply:
we should have integers that big because 10 is the max available capacity here
but I've replaced it with `round` anyway in 2bd967a